lib/connection_pool.rb

```diff
@@ -113,6 +113,13 @@ def shutdown(&block)
     @available.shutdown(&block)
   end
 
+  def stats
```

Comment:
I'm not a fan of this composite stats method.  I'd prefer to see individual methods.